python/pycolfits.py

```diff
@@ -13,10 +13,19 @@ def readfrom(filename, lower_case=False, upper_case=False, **kwargs):
     hdr = hdulist[1].header
     data = hdulist[1].data[0]
     colnames = hdulist[1].data.dtype.names
+    
+    ## check the length of columns to avoid any empty ones
```

Comment:
```suggestion
    # Check the length of columns to avoid any empty ones
```
For consistency with the rest of the code :)